### PR TITLE
Fix nits in comments for NodeResources plugins

### PR DESCRIPTION
**Pull request by chendave**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by alculquicondor** [comment]
Please be more descriptive of your changes (title and description), even if they are simple :)

Something like "Fix nits in comments for NodeResourceFit plugin"

**Comment by alculquicondor** [comment]
Oh and the exact reason is that we used to have scores go from 0 to 10. Now they go from 0 to 100.

**Comment by chendave** [comment]
/hold
waiting for https://github.com/kubernetes/kubernetes/pull/90544

File: pkg/scheduler/framework/plugins/noderesources/least_allocated_test.go [hunk 1]
```diff
@@ -167,8 +167,8 @@ func TestNodeResourcesLeastAllocated(t *testing.T) {
 		},
 		{
 			// Node1 scores on 0-10 scale
-			// CPU Score: ((10000 - 6000) *10) / 10000 = 40
-			// Memory Score: ((20000 - 5000) *10) / 20000 = 75
+			// CPU Score: ((10000 - 6000) *100) / 10000 = 40
```
Comment (alculquicondor) [inline pkg/scheduler/framework/plugins/noderesources/least_allocated_test.go:170]: Can you say MaxNodeScore instead? like the other comment.

**Comment by alculquicondor** [comment]
Not sure why you need to hold on the other PR. This is independent.

File: pkg/scheduler/framework/plugins/noderesources/least_allocated_test.go (hunk 1 above)
Comment (chendave) [reply pkg/scheduler/framework/plugins/noderesources/least_allocated_test.go:170]: sure, sure!

**Comment by chendave** [comment]
If there is no any mistake, I will lift the "hold".

**Pull request merged**